source/template.yaml

```diff
@@ -239,12 +240,38 @@ Resources:
         - IpProtocol: tcp
           FromPort: 443
           ToPort: 443
-          # Should be another security group or VPC CIDR block
-          CidrIp: 0.0.0.0/0
+          SourceSecurityGroupId: !Sub 
+              - '{{resolve:ssm:/${Environment}/network/securitygroup/idp/id:1}}'
+              - Environment: !Ref environment
+
+  # add all functions that need gitsha alias to this list
+  ApplicationLambdaArns:
```

Comment:
Do you think we should try to template this file? There's a lot of copy-paste to add a new lambda